src/main/java/de/qabel/desktop/ui/about/AboutController.java

```diff
@@ -101,6 +113,33 @@ private void indexJarNames() throws ParserConfigurationException, IOException, S
         }
     }
 
+    public void openThanksPopUp(ActionEvent actionEvent) throws IOException{
+        try {
+            InputStream thanksFile = System.class.getResourceAsStream("/files/thanks_file");
+            String contentThanksFile = IOUtils.toString(thanksFile, "UTF-8");
+            showAboutPopUp(contentThanksFile);
+        } catch (IOException e) {
+            e.printStackTrace();
+        } catch (NullPointerException ignored){
+        }
+
+    }
+
+    public void showAboutPopUp(String contentPopup) {
+            stagePopup = new Stage();
+            popupView = new AboutPopupView();
+            scenePopup = new Scene(popupView.getView(),900, 628, true, SceneAntialiasing.BALANCED);
```

Comment:
Actually, the hard coded size does only fit if the user didn't resize the window and the window manager has the "correct" sized borders.
I don't think it needs to be a completely new window. For example, the OfflineIndicator uses such a dark overlay, too and it does it by appending it to the already existing main Scene:
```JAVA
new OfflineView().getViewAsync(window.getChildren()::add);
```